app/src/userpages/components/StreamPage/constants.js

```diff
@@ -11,4 +11,4 @@ export const rightColumn = {
     },
 }
 
-export const fieldTypes = ['number', 'boolean', 'map', 'list', 'string']
+export const fieldTypes = ['number', 'boolean', 'map', 'list', 'string', 'timestamp']
```

Comment:
Should this be `date` instead of `timestamp`?

Reply:
Backend uses `timestamp` as the type identifier.